content/en/docs/reference/command-line-tools-reference/feature-gates/StorageCapacityScoring.md

```diff
@@ -0,0 +1,14 @@
+---
+title: StorageCapacityScoring
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.33"
+---
+Enable the feature that scores nodes based on available
+storage capacity.
```

Comment:
- We are renaming an existing feature gate?
- What kind of storage are we talking about? Local disk? Including ephemeral?
- Which component(s) are affected? Looks like kube-scheduler is unavoidable, any other components affected?

Reply:
I understood. I will add those explanations.

Reply:
I have updated the content to answer your question. Could you please review it again?